### ISSUE TITLE
LuLu blocks youtube-dl Terminal connections

### ISSUE THREAD

**Issue by c3o8k1g** [objective-see/LuLu#195]
I use terminal to perform a number of tasks.
Amongst those, I use [youtube-dl](https://github.com/ytdl-org/youtube-dl/).
Unfortunately, in order to successfully use youtube-dl I have to manually disable LuLu and re-enable it after the fact.
I tried adding rules for every possibility but came up empty.
Any solutions?
Thanks.

**Comment by amnesic** [comment]
Just for the record, I don't have that problem here. I use LuLu with youtube-dl regularly.
(curl & wget is allow on my config )
youtube-dl v.2020.05.03
LuLu v.1.2.3

**Comment by c3o8k1g** [comment]
Thanks for your reply @amnesic!
I must admit I did a very poor job at providing an overview of my issue.

Error: Unable to download webpage: <urlopen error [Errno -536870212] Unknown error: -536870212> (caused by URLError(error(-536870212, 'Unknown error: -536870212'),))
macOS 10.15.2.
LuLu v.1.2.3
I was running youtube-dl v.2020.03.24 installed with curl.

I uninstalled youtube-dl and reinstalled it with wget.
Now running v.2020.05.08, but I get the same error as before.

I strongly suspect the reason to be a setting in LuLu. the reason being every time I disable it, youtube-dl works. If I enable LuLu in the middle of a download, then the download suddenly halts.

Thanks for any help!
